chain/jsonrpc/src/lib.rs

```diff
@@ -211,21 +233,18 @@ impl JsonRpcHandler {
         let result = self
             .client_addr
             .send(NetworkClientMessages::Transaction(tx))
-            .map_err(|err| RpcError::server_error(Some(convert_mailbox_error(err))))
```

Comment:
@bowenwang1996  @frol In what cases RPC will return a `MailboxError`? AFAIU it only happens if the communication between the actors has timed out or the channel has been closed. The latter one can happen when the node is shutting down (btw, we need to implement a way to gracefully shutdown the node). The former one is not supposed to happen in our infra, since the communication between the actors does not have a timeout , correct?

Reply:
Yes I think this error can only happen under extreme circumstances, but @evgenykuzyakov insisted that I handle it. I think it can also happen if the node receives too much traffic.

Reply:
I see. @bowenwang1996 is it correct that communication between our Actix actors uses bounded channels that can overflow and drop the messages? Is it possible to arrive to an unforeseen state due to some messages being lost between the Actix actors? CC @SkidanovAlex

Reply:
I am not sure. I think @frol knows actix better, but we in general should be resistant to lost messages.

Reply:
It looks like Actix uses unbounded channels by the default, so it should be good unless we explicitly changed it. https://github.com/actix/actix/issues/14

Reply:
unbounded channels are the major source of unbounded memory consumption, and they tend to hide problems until it is too late.